src/main/java/com/faforever/client/api/dto/Mod.java

```diff
@@ -35,4 +34,15 @@
 
   @Relationship("latestVersion")
   private ModVersion latestVersion;
+
```

Comment:
The other way arround add @AllArgsConstructor where possible

Reply:
Ok i will take care of it in a few days